### PR TITLE
docs: add controls examples

### PR DESCRIPTION
**Pull request by luwes**
this PR adds the individual controls to the docs site and removes them from the examples site

Preview: https://media-chrome-docs-git-fork-luwes-docs-controls-mux.vercel.app/en/media-play-button

### CODE REVIEW

File: docs/src/pages/en/media-airplay-button.md [hunk 1]
```diff
@@ -2,12 +2,30 @@
 title: <media-airplay-button>
 description: Media Airplay Button
 layout: ../../layouts/MainLayout.astro
+source: https://github.com/muxinc/media-chrome/tree/main/src/js/media-airplay-button.js
```
Comment (gkatsev) [inline docs/src/pages/en/media-airplay-button.md:5]: did we want to add this to the layout to automatically render this link on the pages?

File: docs/src/config.ts [hunk 2]
```diff
@@ -61,20 +61,24 @@ export const SIDEBAR: Sidebar = {
       { text: 'Architecture', link: 'en/architecture-diagrams' },
     ],
     Components: [
+      { text: 'Loading indicator', link: 'en/media-loading-indicator' },
```
Comment (gkatsev) [inline docs/src/config.ts:64]: can we sort these alphabetically? Would be easier to find components.

File: docs/src/pages/en/media-airplay-button.md (hunk 1 above)
Comment (luwes) [reply docs/src/pages/en/media-airplay-button.md:5]: yes I thought to keep this for a next iteration. it's not that important

File: docs/src/config.ts (hunk 2 above)
Comment (luwes) [reply docs/src/config.ts:70]: done

**Pull request merged**